RockPaperScissors/RockPaperScissors.cs

```diff
@@ -1,18 +1,57 @@
 ﻿using System;
 
+
 namespace RockPaperScissors
 {
     class Program
     {
         public static void Main()
         {
-            Console.WriteLine("Enter your hand:");
-            string hand1 = Console.ReadLine().ToLower();
-            string hand2 = ComputerHandGenerator();
-            Console.WriteLine(CompareHands(hand1, hand2));
+            string hand1 = "";
+            bool endGame = false;
+            //Read user's input
+
+
+            //Try-Catch-Finally for error handling
+            while (endGame == false)
+            {
+                Console.WriteLine("Enter your hand:");
+                try
+                {
+                    hand1 = Console.ReadLine().ToLower();
```

Comment:
This will never throw an exception that causes the catch block to run. You need to inspect the `hand1` variable and if it is not rock paper or scissors throw an exception. Then your catch block will run.